drivers/generic_gpio_common.c

```diff
@@ -0,0 +1,444 @@
+/*
+	anything commented is optional
+	anything else is mandatory
+
+	for more information, refer to:
+	* docs/developers.txt
+	* docs/new-drivers.txt
+	* docs/new-names.txt
+
+	and possibly also to:
+	* docs/hid-subdrivers.txt for USB/HID devices
+	* or docs/snmp-subdrivers.txt for SNMP devices
+*/
+/* ./configure --with-pidpath=/run/nut --with-altpidpath=/run/nut --with-statepath=/run/nut --sysconfdir=/etc/nut --with-gpio --with-user=nut --with-group=nut	*/
+#pragma GCC optimize("O0")
```

Comment:
By release time this should go away (or hidden by `#if 0`) :)

Is this to ease debugging, or is there some compile/run-time issue that the driver won't work because of optimizations? Then a comment to keep this pragma long-term would help.